src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -92,6 +92,13 @@ public function load(array $configs, ContainerBuilder $container)
             $dir = dirname($reflection->getFileName()).'/Resources/views';
             if (is_dir($dir)) {
                 $this->addTwigPath($twigFilesystemLoaderDefinition, $dir, $bundle);
+
+                $classMock = new $class();
+                $parent = $classMock->getParent();
+
+                if ($parent) {
```

Comment:
Yes, with performance in mind. Each bundle in the map should only be initialized once. Just keep in mind you're in a loop.